mono/mini/mini-runtime.c

```diff
@@ -3693,8 +3693,6 @@ mini_parse_debug_option (const char *option)
 		mini_debug_options.gdb = TRUE;
 	else if (!strcmp (option, "lldb"))
 		mini_debug_options.lldb = TRUE;
-	else if (!strcmp (option, "llvm-disable-self-init"))
```

Comment:
Could still accept it in parsing, but warn and do nothing, if there is a compat concern.

Reply:
Its broken, so unlikely that anyone is using it.